modules/src/jokes.txt

```diff
@@ -0,0 +1,24 @@
+'''
+This file contains of all jokes to be served by JARVIS
+
+Format:
+
+<question>\t\t\t<answer>\n
+             (OR)
+<entire_joke_here>\n
+
+The "\t\t\t" splits the joke as question & answer( if its Q&A type) and \n is used for new line.
```

Comment:
1. Every joke is split by a \n
2. Inside the joke string, it can be any model like a classic Q/A model or a one-liner.
3. In the final bot, question and answer will be clubbed as one with a \t\t\t where [0]th element of this split will be question and [1] would be the answer.